src/api/db/migrate/20211214111900_add_scm_elements.rb

```diff
@@ -0,0 +1,6 @@
+class AddScmElements < ActiveRecord::Migration[6.1]
```

Comment:
This would be more descriptive:

```suggestion
class AddScmsyncToPackagesAndProjects< ActiveRecord::Migration[6.1]
```

The file should also be renamed to follow this change.